pkg/util/helper/binding.go

```diff
@@ -68,6 +68,81 @@ func (p ClusterWeightInfoList) GetWeightSum() int64 {
 	return res
 }
 
+// Dispenser aims to divide replicas among clusters by different weights.
+type Dispenser struct {
+	// Target replicas, should be a positive integer.
+	NumReplicas int32
+	// Final result.
+	Result []workv1alpha2.TargetCluster
+}
+
+// NewDispenser will construct a dispenser will target replicas and a prescribed initial result.
```

Comment:
NewDispenser will construct a dispenser with target replicas and a prescribed initial result?